frontend/awx/resources/inventories/hooks/useInventoriesSourcesToolbarActions.tsx

```diff
@@ -61,7 +88,47 @@ export function useInventoriesSourcesToolbarActions(view: IAwxView<InventorySour
         isDanger: true,
         isDisabled: (sources: InventorySource[]) => cannotDeleteResources(sources, t),
       },
+      {
+        type: PageActionType.Button,
```

Comment:
I know we have this as a separate button in old UI, but maybe it would be more consistent with framework patterns if we put it in the bulk actions kebab menu? @trahman73 @tiyiprh what do ya'll think?

Reply:
I think that makes sense!

Reply:
I can do that, but we have plenty of buttons outside of kebab. It depends on how frequently will user use this option, clicking kebab is additional click, which is unecessary... Kebab menu makes sense when there is no space for all buttons, here, we have plenty of space.

Reply:
Do you all know if syncing is common for users? If so, agree that it can be outside of the kebab but if it's not common for users to sync, it should remain in the kebab.

Reply:
It probably isn't super common to sync from the list.  it is probably much more common to sync an inventory source from within a workflow job template...which doesn't involve the list.  Thus I think putting the button in the kebab on the toolbar would be fine.